src/components/CentralContainer/Sidebar/SidebarTabs/SearchTabPanel/index.tsx

```
@@ -0,0 +1,109 @@
+                                value={SEARCH_OPTION.IS_CASE_SENSITIVE}
+                            >
+                                Aa
+                            </IconButton>
+                            <IconButton
+                                sx={{fontFamily: "Inter"}}
+                                value={SEARCH_OPTION.IS_REGEX}
+                            >
+                                .*
+                            </IconButton>
+                        </ToggleButtonGroup>
+                    </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)